build.sh

```diff
@@ -57,6 +57,24 @@ do
   esac
 done
 
+if [ "$DO_UPDATE" == true ] && [ "$QUIET" == false ]; then
+  echo "Checking if html-build is up to date..."
+  ORIGIN_URL=$(git remote get-url origin)
+  git fetch -q $ORIGIN_URL master
```

Comment:
Eh, I guess it's fine to leave it for them to discover on their own when they do the pull or whatever. I was thinking it might be nice to explain why we're erroring out---what critical functionality have you missed out on. But no big deal.